backend_1/b/b/.gitignore

```diff
@@ -0,0 +1,25 @@
+# Logs
```

Comment:
Folder structure should be updated to have frontend and backend in the same level.